src/base/RunnersRegistry.h

```diff
@@ -33,7 +33,10 @@
  */
 
 /// a runnable registrant API
-/// kids must override [only] the methods they are interested in
+/// Kids must override [only] the methods they are interested in.
+/// Runners operate with low privilege level, methods needing
+/// high privilege must ensure they have reduced privileges
+/// before returning.
```

Comment:
"Runners operate with low privileges" is documenting how runners operate -- something we do not (and cannot) control. What we do control (and should document) is event preconditions and postconditions:

```suggestion
/// Kids must override [only] the methods handling the events they are interested in.
/// All events are triggered in low privilege level context.
/// All overridden methods must exit in low privilege level context.
```

The above also avoids using the `return` keyword that might be misinterpreted as placing exceptions outside this postcondition requirement.

I am not satisfied with the details of my suggested wording, but do not want to spend more time on this until we finalize which events these pre/postconditions apply to.

P.S. I doubt the underlying "all events" assumption survives for much longer due to concerns expressed in another change request, but this change request can be easily applied to any final SUID-related documentation in the RegisteredRunner class context.